packages/core/content-type-builder/server/src/services/architect.ts

```diff
@@ -166,7 +201,7 @@ export const create = async (prompt: string, previousSchema?: object) => {
         singularName: z.string(),
         pluralName: z.string(),
         description: z.string().optional(),
-        name: z.string(),
+        name: z.string().optional(), // Model forgets to add this sometimes
```

Comment:
LLMs are dumb :man_shrugging:

Reply:
hahaha yes they are,
we need to adapt the prompt for it to be reliable, and the best approach imo is to just ask for dumb things, and make it spit dumb plain jsons, and avoid nested keys if possible